### ISSUE TITLE
Hekili\Events.lua:1764: attempt to perform arithmetic on a nil value (SpellQueueWindow)

### ISSUE THREAD

**Issue by Aravj** [Hekili/hekili#524]
Got a couple of errors when i login. 

Strange thing is, i can get it to work if i delete the profile and reinstall the addon. But that's more a 50/50 kind of thing.

Last working Version was "v9.0.1-1.0.18-beta1". Don't know if this is important, but i did a new Installation of WoW and changed the language from english to german. 
--------------------------------------------
First Error:
```
Hekili\Events.lua:1764: attempt to perform arithmetic on a nil value
[string "@Hekili\Events.lua"]:1764: in main chunk

Locals:
addon = "Hekili"
ns = <table> {
 getLocalClass = <function> defined @Hekili\Constants.lua:9
 hasRequiredResources = <function> defined @Hekili\State.lua:6170
 auras = <table> {
 }
 getSpecializationKey = <function> defined @Hekili\Constants.lua:211
 forecastResources = <function> defined @Hekili\State.lua:1134
 frameProfile = <table> {
 }
 UnregisterUnitEvent = <function> defined @Hekili\Events.lua:172
 debug = <table> {
 }
 GetTexString = <function> defined @Hekili\Utils.lua:361
 queue = <table> {
 }
 updateConduits = <function> defined @Hekili\Events.lua:1603
 castsAll = <table> {
 }
 cpuProfile = <table> {
 }
 GroupMembers = <function> defined @Hekili\Utils.lua:247
 spendResources = <function> defined @Hekili\State.lua:5797
 safeMax = <function> defined @Hekili\Utils.lua:234
 addMetaFunction = <function> defined @Hekili\State.lua:1529
 targets = <table> {
 }
 metatables = <table> {
 }
 IsActiveSpell = <function> defined @Hekili\Utils.lua:312
 keys = <table> {
 }
 addCooldownMetaFunction = <function> defined @Hekili\State.lua:2509
 attr = <table> {
 }
 StartConduits = <function> defined @Hekili\Events.lua:1660
 RegisterEvent = <function> defined @Hekili\Events.lua:103
 updateEssences = <function> defined @Hekili\Events.lua:607
 escapeMagic = <function> defined @Hekili\Utils.lua:101
 StopEventHandler = <function> defined @Hekili\Events.lua:92
 SpaceOut = <function> defined @Hekili\Utils.lua:50
 roundUp = <function> defined @Hekili\Utils.lua:130
 getClassID = <function> defined @Hekili\Constants.lua:40
 ResourceRegenerates = <function> defined @Hekili\Constants.lua:99
 fsub = <function> defined @Hekili\Utils.lua:96
 GetAtlasCoords = <function> defined @Hekili\Utils.lua:382
 titlefy = <function> defined @Hekili\Utils.lua:87
 updatePowers = <function> defined @Hekili\Events.lua:492
 FrameStratas = <table> {
 }
 round = <function> defined @Hekili\Utils.lua:123
 GetResourceKey = <function> defined @Hekili\Constants.lua:88
 GetResourceID = <function> defined @Hekili\Constants.lua:83
 getSpecializationID = <function> defined @Hekili\Constants.lua:216
 updateGear = <function> defined @Hekili\Events.lua:998
 UI = <table> {
 }
 Error = <function> defined @Hekili\Utils.lua:15
 multiUnpack = <function> defined @Hekili\Utils.lua:108
 FindUnitBuffByID = <function> defined @Hekili\Utils.lua:280
 orderedPairs = <function> defined @Hekili\Utils.lua:217
 CachedGetItemInfo = <function> defined @Hekili\Utils.lua:339
 PTR = false
 formatValue = <function> defined @Hekili\Formatting.lua:774
 castsOn = <table> {
 }
 TTD = <table> {
 }
 StartEventHandler = <function> defined @Hekili\Events.lua:76
 castsOff = <table> {
 }
 updateTalents = <function> defined @Hekili\Events.lua:417
 safeMin = <function> defined @Hekili\Utils.lua:222
 RegisterUnitEvent = <function> defined @Hekili\Events.lua:133
 getInverseDirection = <function> defined @Hekili\Constants.lua:25
 Patrons = <table> {
 }
 hotkeys = <table> {
 }
 FeignEvent = <function> defined @Hekili\Events.lua:185
 lib = <table> {
 }
 snapshots = <table> {
 }
 GetResourceInfo = <function> defined @Hekili\Constants.lua:78
 Tooltip = HekiliTooltip {
 }
 formatKey = <function> defined @Hekili\Utils.lua:63
 commitKey = <function> defined @Hekili\Utils.lua:163
 titleCase = <function> defined @Hekili\Utils.lua:68
 FindU
```
-----------------------------------------
Second Error:
```
Hekili\Core.lua:188: attempt to call field 'ReadKeybindings' (a nil value)
[string "@Hekili\Core.lua"]:188: in function <Hekili\Core.lua:184>
[string "=[C]"]: ?
[string "@ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:615>
```
--------------------------------
Third Error:
```
Hekili\Classes.lua:5388: attempt to call method 'UpdateDamageDetectionForCLEU' (a nil value)
[string "@Hekili\Classes.lua"]:5388: in function `SpecializationChanged'
[string "@Hekili\Events.lua"]:336: in function `handler'
[string "@Hekili\Events.lua"]:56: in function <Hekili\Events.lua:50>

Locals:
self = <table> {
 GetTTD = <function> defined @Hekili\Targets.lua:839
 DeserializeStyle = <function> defined @Hekili\Options.lua:8892
 modules = <table> {
 }
 EmbedAbilityOptions = <function> defined @Hekili\Options.lua:3634
 Scripts = <table> {
 }
 GetDeathClockByGUID = <function> defined @Hekili\Targets.lua:824
 GetPetBasedTargetSpells = <function> defined @Hekili\Targets.lua:88
 After = <function> defined @Hekili\Utils.lua:268
 NewSpellInfo = true
 Run = <function> defined @Hekili\Hekili.lua:263
 UpdateDisplayVisibility = <function> defined @Hekili\UI.lua:1630
 IsEnabled = <function> defined @ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:451
 RemoveHold = <function> defined @Hekili\State.lua:5862
 GetPetBasedTargetSpell = <function> defined @Hekili\Targets.lua:100
 ToggleHold = <function> defined @Hekili\State.lua:5887
 SpecializationChanged = <function> defined @Hekili\Classes.lua:5122
 UpdateUseItems = <function> defined @Hekili\Events.lua:646
 SetSpecOption = <function> defined @Hekili\Options.lua:3546
 GetDefaults = <function> defined @Hekili\Options.lua:530
 dumpKeyCache = <table> {
 }
 SetEnabledState = <function> defined @ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:424
 GetDisplayByName = <function> defined @Hekili\Core.lua:1283
 Options = <table> {
 }
 ReInitialize = <function> defined @Hekili\Core.lua:161
 npUnits = <table> {
 }
 GetAbilityInfo = <function> defined @Hekili\Classes.lua:5069
 State = <table> {
 }
 OnEnable = <function> defined @Hekili\Core.lua:184
 RestoreDefault = <function> defined @Hekili\Classes.lua:794
 CmdLine = <function> defined @Hekili\Options.lua:8630
 IsDisplayActive = <function> defined @Hekili\UI.lua:1727
 SetupPetBasedTargetDetection = <function> defined @Hekili\Targets.lua:114
 ShowDiagnosticTooltip = <function> defined @Hekili\UI.lua:2315
 Notify = <function> defined @Hekili\Options.lua:9532
 OnInitialize = <function> defined @Hekili\Core.lua:61
 NewModule = <function> defined @ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:235
 GetErrors = <function> defined @Hekili\Utils.lua:41
 DumpPetBasedTargetInfo = <function> defined @Hekili\Targets.lua:140
 SetPackShareOption = <function> defined @Hekili\Options.lua:3530
 CreateButton = <function> defined @Hekili\UI.lua:1765
 ReleaseHolds = <function> defined @Hekili\State.lua:5919
 Deserialize = <function> defined @BugSack\Libs\AceSerializer-3.0\AceSerializer-3.0.lua:246
 SaveDebugSnapshot = <function> defined @Hekili\Hekili.lua:332
 SetSpecPref = <function> defined @Hekili\Options.lua:3587
 GetName = <function> defined @ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:279
 RunSpellCallbacks = <function> defined @Hekili\Events.lua:280
 GetSpec = <function> defined @Hekili\Classes.lua:5392
 SerializeDisplay = <function> defined @Hekili\Options.lua:8789
 name = "Hekili"
 ExpireTTDs = <function> defined @Hekili\Targets.lua:1029
 GetEncounterList = <function> defined @Hekili\Options.lua:72
 NewSpecialization = <function> defined @Hekili\Classes.lua:836
 Disable = <function> defined @ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:314
 GetUnitByName = <function> defined @Hekili\Targets.lua:51
 auditInterval = 0
 GetNextPrediction = <function> defined @Hekili\Core.lua:1200
 currentSpec = <table> {
 }
 Print = <function> defined @Interfac
```
-------------------------------------------------------------------------------------------------------
Fourth Error:
```
Hekili\Core.lua:1594: attempt to call method 'GetBindingForAction' (a nil value)
[string "@Hekili\Core.lua"]:1594: in function `ProcessHooks'
[string "@Hekili\UI.lua"]:826: in function <Hekili\UI.lua:690>

Locals:
self = <table> {
 GetTTD = <function> defined @Hekili\Targets.lua:839
 DeserializeStyle = <function> defined @Hekili\Options.lua:8892
 modules = <table> {
 }
 EmbedAbilityOptions = <function> defined @Hekili\Options.lua:3634
 Scripts = <table> {
 }
 GetDeathClockByGUID = <function> defined @Hekili\Targets.lua:824
 GetPetBasedTargetSpells = <function> defined @Hekili\Targets.lua:88
 After = <function> defined @Hekili\Utils.lua:268
 NewSpellInfo = true
 Run = <function> defined @Hekili\Hekili.lua:263
 UpdateDisplayVisibility = <function> defined @Hekili\UI.lua:1630
 IsEnabled = <function> defined @Details\Libs\AceAddon-3.0\AceAddon-3.0.lua:451
 RemoveHold = <function> defined @Hekili\State.lua:5862
 GetPetBasedTargetSpell = <function> defined @Hekili\Targets.lua:100
 ToggleHold = <function> defined @Hekili\State.lua:5887
 SpecializationChanged = <function> defined @Hekili\Classes.lua:5122
 UpdateUseItems = <function> defined @Hekili\Events.lua:646
 SetSpecOption = <function> defined @Hekili\Options.lua:3546
 GetDefaults = <function> defined @Hekili\Options.lua:530
 dumpKeyCache = <table> {
 }
 SetEnabledState = <function> defined @Details\Libs\AceAddon-3.0\AceAddon-3.0.lua:424
 GetDisplayByName = <function> defined @Hekili\Core.lua:1283
 Options = <table> {
 }
 ReInitialize = <function> defined @Hekili\Core.lua:161
 npUnits = <table> {
 }
 GetAbilityInfo = <function> defined @Hekili\Classes.lua:5069
 State = <table> {
 }
 OnEnable = <function> defined @Hekili\Core.lua:184
 RestoreDefault = <function> defined @Hekili\Classes.lua:794
 CmdLine = <function> defined @Hekili\Options.lua:8630
 IsDisplayActive = <function> defined @Hekili\UI.lua:1727
 SetupPetBasedTargetDetection = <function> defined @Hekili\Targets.lua:114
 ShowDiagnosticTooltip = <function> defined @Hekili\UI.lua:2315
 Notify = <function> defined @Hekili\Options.lua:9532
 OnInitialize = <function> defined @Hekili\Core.lua:61
 NewModule = <function> defined @Details\Libs\AceAddon-3.0\AceAddon-3.0.lua:235
 GetErrors = <function> defined @Hekili\Utils.lua:41
 DumpPetBasedTargetInfo = <function> defined @Hekili\Targets.lua:140
 SetPackShareOption = <function> defined @Hekili\Options.lua:3530
 CreateButton = <function> defined @Hekili\UI.lua:1765
 ReleaseHolds = <function> defined @Hekili\State.lua:5919
 Deserialize = <function> defined @BugSack\Libs\AceSerializer-3.0\AceSerializer-3.0.lua:246
 SaveDebugSnapshot = <function> defined @Hekili\Hekili.lua:332
 SetSpecPref = <function> defined @Hekili\Options.lua:3587
 GetName = <function> defined @Details\Libs\AceAddon-3.0\AceAddon-3.0.lua:279
 RunSpellCallbacks = <function> defined @Hekili\Events.lua:280
 GetSpec = <function> defined @Hekili\Classes.lua:5392
 SerializeDisplay = <function> defined @Hekili\Options.lua:8789
 name = "Hekili"
 ExpireTTDs = <function> defined @Hekili\Targets.lua:1029
 GetEncounterList = <function> defined @Hekili\Options.lua:72
 NewSpecialization = <function> defined @Hekili\Classes.lua:836
 Disable = <function> defined @Details\Libs\AceAddon-3.0\AceAddon-3.0.lua:314
 GetUnitByName = <function> defined @Hekili\Targets.lua:51
 auditInterval = 0
 GetNextPrediction = <function> defined @Hekili\Core.lua:1200
 currentSpec = <table> {
 }
 Print = <function> defined @AstralKeys\Libs\AceConsole-3.0\
```

**Comment by Hekili** [comment]
Interesting, it's probably related to how long it takes your game to load.  Only the first error would actually matter.

**Comment by Aravj** [comment]
It takes about 30 seconds to login and the bugsack error sound about half way in ^^

**Comment by Radius256** [comment]
I'm getting these errors too.  I've tried disabling all other addons.  Taking everything off my bars and then putting them back on.  Resetting my keybinds. 
I installed a clean install of wow, on a different computer, and was still getting these errors.
When I first log in, it is fine.  Shows 3 (default) Hekili icons.  If I log out and back in, or /reload, I get these errors and I only see 1 Hekili icon, or none.  Doing it on all my characters, warrior, hunter, priest, DH

Thanks

**Comment by Hekili** [comment]
Could you post snapshots?

**Comment by Radius256** [comment]
![WoWScrnShot_110420_182649](https://user-images.githubusercontent.com/73975535/98190170-5056eb00-1ecb-11eb-8f9d-02f10d2dce37.jpg)

**Comment by Hekili** [comment]
No, see /hekili > Snapshots

**Comment by Radius256** [comment]
Click here and press CTRL+A, CTRL+C to copy the snapshot.

build: v9.0.1-1.0.21
level: 50 (50)
class: WARRIOR
spec: protection

talents: anger_management
    best_served_cold
    bounding_stride
    devastator
    double_time
    indomitable
    unstoppable_force

pvptalents: none

covenant: none

azerite: none

essences: [none], none

sets/legendaries/artifacts: none

gear: none

legendaries: none

itemIDs: none

settings: aoe = 2
    buffPadding = 0
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = false
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 8
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    gcdSync = true
    maxRefresh = 10
    maxTime = 33
    nameplateRange = 8
    nameplates = true
    package = Protection Warrior
    petbased = false
    potion = potion_of_unbridled_fury
    potionsReset = 20180919.1
    throttleRefresh = false
    throttleTime = false
    free_revenge = true

toggles: cooldowns = false
    custom1 = false
    custom2 = false
    defensives = false
    essences = true
    interrupts = false
    mode = automatic
    potions = false

New Recommendations for [ Defensives ] requested at 18:35:57 ( 980688.98 ); using built-in ( Protection Warrior ) priority.

RECOMMENDATION #1 ( Offset: 0.00, GCD: 0.00, Casting: 0.00 ).

    Resources: rage[ 0.00 / 100.00 ]
    Checking if I'm casting ( No ) and if it is a channel ( No ).
    No whitelist.
    
    Processing precombat action list [ Protection Warrior - precombat ].
        
        Current recommendation was NO ACTION at +60.00s.
        
        1.   battle_shout ( precombat - 1 ) - ability disabled ( display )
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-precombat-1:battle_shout:0.02
        
        2.   azsharas_font_of_power ( precombat - 2 ) - IsUsableItem
        Time spent on this action:  0.05ms
        TimeData:Protection Warrior-precombat-2:azsharas_font_of_power:0.05
        
        3.   worldvein_resonance ( precombat - 3 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-precombat-3:worldvein_resonance:0.02
        
        4.   memory_of_lucid_dreams ( precombat - 4 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-precombat-4:memory_of_lucid_dreams:0.01
        
        5.   potion ( precombat - 5 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-precombat-5:potion:0.01
    
    Completed precombat action list [ Protection Warrior - precombat ].
    
    Processing default action list [ Protection Warrior - default ].
        
        Current recommendation was NO ACTION at +60.00s.
        
        6.   charge ( default - 1 ) - ability disabled ( display )
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-1:charge:0.02
        
        7.   pummel ( default - 2 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-2:pummel:0.01
        
        8.   shockwave ( default - 3 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-3:shockwave:0.01
        
        9.   blood_fury ( default - 4 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-4:blood_fury:0.02
        
        10.  berserking ( default - 5 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-5:berserking:0.02
        
        11.  arcane_torrent ( default - 6 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-6:arcane_torrent:0.02
        
        12.  lights_judgment ( default - 7 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-7:lights_judgment:0.01
        
        13.  fireblood ( default - 8 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-8:fireblood:0.01
        
        14.  ancestral_call ( default - 9 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-9:ancestral_call:0.01
        
        15.  bag_of_tricks ( default - 10 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-10:bag_of_tricks:0.01
        
        16.  potion ( default - 11 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-11:potion:0.01
        
        17.  ignore_pain ( default - 12 ) - ability disabled ( toggle )
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-12:ignore_pain:0.02
        
        18.  worldvein_resonance ( default - 13 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-13:worldvein_resonance:0.01
        
        19.  memory_of_lucid_dreams ( default - 14 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-14:memory_of_lucid_dreams:0.01
        
        20.  concentrated_flame ( default - 15 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-15:concentrated_flame:0.01
        
        21.  heart_essence ( default - 16 ) - IsUsableItem
        Time spent on this action:  0.09ms
        TimeData:Protection Warrior-default-16:heart_essence:0.09
        
        22.  use_items ( default - 17 )
        Criteria for ??? FAIL at +0.00 - cooldown.avatar.remains[90.00] <= gcd.execute[0.00] | buff.avatar.up[false]
        
        23.  avatar ( default - 18 ) - ability disabled ( display )
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-18:avatar:0.02
        
        24.  spear_of_bastion ( default - 19 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-19:spear_of_bastion:0.02
        
        25.  conquerors_banner ( default - 20 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-default-20:conquerors_banner:0.02
        
        26.  ancient_aftershock ( default - 21 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-default-21:ancient_aftershock:0.01
        
        27.  call_action_list ( default - 22 )
        Criteria for mitigation FAIL at +0.00 - incoming_damage_5s[0.00] > 0
        
        28.  run_action_list ( default - 23 )
        Criteria for aoe FAIL at +0.00 - spell_targets.thunder_clap[1.00] >= 3
        
        29.  call_action_list ( default - 24 )
        There is no criteria for st.
        Action list (st) was found.
        + st
        Current recommendation was NO ACTION at +60.00s.
        
        30.  thunder_clap ( st - 1 ) - ability disabled ( display )
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-st-1:thunder_clap:0.02
        
        31.  shield_block ( st - 2 ) - ability disabled ( toggle )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-2:shield_block:0.01
        
        32.  shield_slam ( st - 3 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-3:shield_slam:0.01
        
        33.  thunder_clap ( st - 4 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-4:thunder_clap:0.01
        
        34.  demoralizing_shout ( st - 5 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-5:demoralizing_shout:0.01
        
        35.  ashvanes_razor_coral ( st - 6 ) - IsUsableItem
        Time spent on this action:  0.03ms
        TimeData:Protection Warrior-st-6:ashvanes_razor_coral:0.03
        
        36.  ashvanes_razor_coral ( st - 7 ) - IsUsableItem
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-7:ashvanes_razor_coral:0.01
        
        37.  dragon_roar ( st - 8 ) - talent [ dragon_roar ] missing
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-8:dragon_roar:0.01
        
        38.  thunder_clap ( st - 9 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-9:thunder_clap:0.01
        
        39.  revenge ( st - 10 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-10:revenge:0.01
        
        40.  grongs_primal_rage ( st - 11 ) - IsUsableItem
        Time spent on this action:  0.03ms
        TimeData:Protection Warrior-st-11:grongs_primal_rage:0.03
        
        41.  ravager ( st - 12 ) - talent [ ravager ] missing
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-12:ravager:0.01
        
        42.  condemn ( st - 13 ) - ability unknown
        Time spent on this action:  0.02ms
        TimeData:Protection Warrior-st-13:condemn:0.02
        
        43.  execute ( st - 14 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-14:execute:0.01
        
        44.  victory_rush ( st - 15 ) - ability disabled ( display )
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-15:victory_rush:0.01
        
        45.  impending_victory ( st - 16 ) - talent [ impending_victory ] missing
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-16:impending_victory:0.01
        
        46.  devastate ( st - 17 ) - talent [ devastator ] disallowed
        Time spent on this action:  0.01ms
        TimeData:Protection Warrior-st-17:devastate:0.01
        Returned from list (st), current recommendation is NO ACTION (+60.00).
        - st
    
    Completed default action list [ Protection Warrior - default ].
    Recommendation is NO ACTION at 0.00 + 60.00.
    Recommendation #1 is NO ACTION at 60.00s (60.00s).
    Time spent generating recommendations:  1.54ms

**Comment by Radius256** [comment]
That was while it was in it's errored state, like the screenshot above

**Comment by Hekili** [comment]
I'm testing a fix that looks like it'll be in the next build.

**Comment by Radius256** [comment]
awesome, thanks

**Comment by Hekili** [comment]
Should be resolved in 1.0.22; let me know if I'm wrong.
